nuki/__init__.py

```diff
@@ -80,14 +81,15 @@ def handle_connection(self):
 
 
 class Nuki(SmartPlugin):
-    PLUGIN_VERSION = "1.5.0.7"
+    PLUGIN_VERSION = "1.5.0.6"
```

Comment:
Any serious reason why you start decrement the version number? And if you increase this number please mind to increase the version number in plugin.yaml, too.

Reply:
sorry, cut & paste without thinking about it will kill us all sometime ;-)
This can be changed to 1.5.0.8